autoparallel/_passes/graph_multiplex.py

```diff
@@ -79,25 +77,20 @@ def multiplex_fw_bw_graph(
             old_node_to_new_node[n] = new_node
             insert_point = new_node
 
-    assert bw_graph_op_node is not None
-    assert multiplexed_graph_op_node is not None
-
     # Collect output arguments from backward graph, remapping to new nodes
     bw_op_node_args = [
         old_node_to_new_node[n] if n is not None else None
         for n in bw_graph_op_node.args[0]
     ]
 
-    # Collect output arguments from forward graph
+    # Collect output arguments from multiplexed graph (will contain only fwd_outs)
+    multiplexed_graph_outputs = multiplexed_gm.graph.find_nodes(op="output")
+    assert len(multiplexed_graph_outputs) == 1
```

Comment:
why this assert? is this a tuple of forward outs or are you hardcoding the model arch into the infra?